docs/src/developers/roadmap.md

```diff
@@ -25,10 +25,6 @@ represent broad themes that we see as areas in which JuMP could be improved.
    [https://github.com/jump-dev/JuMP.jl/issues/2227](https://github.com/jump-dev/JuMP.jl/issues/2227)
    JuMP has a strong focus on linear, conic and nonlinear optimization problems.
    We want to add better support for constraint programming.
- - Add support for multiobjective problems:
```

Comment:
Maybe strike-through or add a note that it's done?

Reply:
Changed to strike-through

Reply:
Hmm didn't work: https://jump.dev/JuMP.jl/previews/PR3223/developers/roadmap/. I'll have to find the syntax for documenter